client/helpers/scenes.ts

```diff
@@ -136,35 +117,47 @@ export async function createSceneAndSceneItems({
   onMsg = console.log,
   onProgress,
   parallelism,
-  polling,
+  polling = { intervalMs: PollIntervalMs, maxAttempts: MaxAttempts },
   verbose,
 }: CreateSceneAndSceneItemsReq): Promise<CreateSceneAndSceneItemsRes> {
   const scene = (
     await client.scenes.createScene({ createSceneRequest: createSceneReq() })
   ).data;
   const sceneId = scene.data.id;
+  const res = await createSceneItems({
+    client,
+    createSceneItemReqs,
+    failFast: failFast ?? false,
+    onProgress,
+    parallelism,
+    sceneId,
+  });
   const { a: queuedItems, b: errors } = partition(
-    (
-      await createSceneItems({
-        client,
-        createSceneItemReqs,
-        failFast: failFast ?? false,
-        onProgress,
-        parallelism,
-        sceneId,
-      })
-    ).queuedSceneItems,
-    (i: QueuedSceneItem) => !nullOrUndefined(i.job)
+    res.queuedSceneItems,
+    (i: QueuedSceneItem) => isQueuedJob(i.res)
   );
 
-  if (queuedItems.length === 0) return { errors, scene };
+  if (queuedItems.length === 0 || errors.length === res.leaves)
+    return { errors, scene };
 
-  await pollQueuedJob<SceneItem>({
-    id: (queuedItems[queuedItems.length - 1].job as QueuedJob).data.id,
-    getQueuedJob: (id) => client.sceneItems.getQueuedSceneItem({ id }),
-    allow404: true,
-    polling,
-  });
+  const limit = pLimit(parallelism);
+  await Promise.all(
```

Comment:
Instead of just waiting for the last one we sent, wait on all of them.

Reply:
Just so I'm clear - if we queue up 50k items, we'll poll all 50k until done? Also, if this is the case, we'll probably want to communicate with existing clients that the perceived time to create the scene may increase.

Reply:
Yes, we'll poll 50k, but only `parallelism` at a time, which defaults to 20. So I would imagine most will only be polled once. But yes, this also means scenes will seem to take longer to complete from the client's perspective.